rego/rego.go

```diff
@@ -342,6 +343,12 @@ func EvalPrintHook(ph print.Hook) EvalOption {
 	}
 }
 
+func EvalVirtualCache(vc topdown.VirtualCache) EvalOption {
```

Comment:
A comment here to explain the intended, long running, debugger use case would be helpful here I think. Just in case it's interpreted as a performance thing or similar.